src/main.rs

```diff
@@ -130,6 +130,34 @@ fn main() -> io::Result<()> {
         );
     }
 
+    // Copy or create favicon.ico
+    let favicon_dst = output_folder.join("favicon.ico");
+
+    // Possible paths where favicon.ico might exist
+    let favicon_src_paths = [
+        input_folder.join("static").join("favicon.ico"),           // User's favicon.ico
+        Path::new("example/static/favicon.ico").to_path_buf(),     // Default favicon.ico in project
+    ];
```

Comment:
This path assumes `marmite` will always execute from the repository rootm but
when user installs from `cargo install marmite` or downloads the binary directly, they will have only the `marmite` binary, and optionally the `input_folder/static/favicon.ico`.

On #20 I want to embed all the essential files from `example` in the binary, for now we may have only one possible favicon location: 

```suggestion
    let favicon_src_paths = [
        input_folder.join("static").join("favicon.ico"),           // User's favicon.ico
        // on #20 we may have embedded statics
    ];
```

Reply:
I hadn't paind attention on that, you`re right. I'll think of something better to resolve both issues.